Source/JavaScriptCore/JavaScriptCore.xcodeproj/project.pbxproj

```diff
@@ -13186,6 +13169,7 @@
 			buildActionMask = 2147483647;
 			files = (
 				44F93E0E2AE71FBD00FFA37C /* JavaScriptCoreFramework.cpp in Sources */,
+				FFF3372F2BCDE4A40070D04B /* JSCBytecodeCacheVersion.cpp in Sources */,
```

Comment:
This looks very strange. I think probably this is not a right place.

Reply:
I recommend first completely removing  `JSCBytecodeCacheVersion.cpp` etc. from Xcode, and just re-add them as a normal files.